pyaedt/hfss.py

```diff
@@ -1143,7 +1154,7 @@ def create_sbr_linked_antenna(
              The default is ``"0.95"``.
         visible : bool, optional.
             Visualize source objects in target design. The default is ``True``.
```

Comment:
```suggestion
            Whether to visualize source objects in the target design. The default is ``True``.
```
Is "to visualize" correct--or should it be something like this: "Whether to make source objects in the target design visible."

Reply:
The second one